Updating BCs Documentation (#250)
shannon-lab/zapdos: doc/content/source/bcs/SecondaryElectronEnergyBC.md

# SecondaryElectronEnergyBC

> !syntax description /BCs/SecondaryElectronEnergyBC

Reviewer: No class description

## Overview

> Note: add $n_{\gamma}$

Reviewer: This should be addressed but again I have no idea what this means

`SecondaryElectronEnergyBC` assumes the electrostatic approximation for the electric field.

\begin{equation}
a =

> \begin{cases}
> 1, & \mu_{e} \left( \nabla (V) \right) \cdot \textbf{n} > 0\\
> 0, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \leq 0\\
> \end{cases} \\[10pt]

Reviewer: ```suggestion
\begin{cases}
1, & \mu_{e} \left( -  \nabla (V) \right) \cdot \textbf{n} > 0\\
0, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \leq 0\\
\end{cases} \\[10pt]
```

I wasn't sure if leaving it off was intentional or not. If it is intentional just ignore this.

> \begin{equation}
> a =
> \begin{cases}
> 1, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} > 0\\
> 0, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \leq 0\\
> \end{cases} \\[10pt]
> \Gamma_{\epsilon} \cdot \textbf{n} = \frac{5}{3} \frac{-2}{1+r} (1 - a_{e}) \gamma \Gamma_{j} \cdot \textbf{n} \\[10pt]
> \Gamma_{j} \cdot \textbf{n} = \frac{1-r_{j}}{1+r_{j}} \left[ (2 a_{j}-1) \ \mu_{j} \text{-} \nabla (V) \cdot \textbf{n} \ n_{j} + \frac{1}{2}v_{th,j}n_{j} \right] \\[10pt]
> v_{th,j} = \sqrt{\frac{8k_{B}T}{\pi m_{j}}}
> \end{equation}

Reviewer: ```suggestion
\begin{equation}
a =
\begin{cases}
1, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} > 0\\
0, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \leq 0\\
\end{cases} \\[10pt]
\Gamma_{\epsilon} \cdot \textbf{n} = \frac{5}{3} \frac{-2}{1+r} (1 - a_{e}) \gamma \Gamma_{j} \cdot \textbf{n} \\[10pt]
\Gamma_{j} \cdot \textbf{n} = \frac{1-r_{j}}{1+r_{j}} \left[ (2 a_{j}-1) \ \mu_{j}
\left(  \text{-} \nabla (V) \right) 
\cdot \textbf{n} \ n_{j} + \frac{1}{2}v_{\text{th},j}n_{j} \right] \\[10pt]
v_{\text{th},j} = \sqrt{\frac{8k_{B}T}{\pi m_{j}}}
\end{equation}
```

> Where $\Gamma$ is the flux of the electron or ions, $\textbf{n}$ is the normal of the boundary,
> $\mu$ is the mobility coefficient, $n$ is the specie density, $v_{th}$ is the thermal velocity of the species, $k_{B}$ is the Boltzmann constant, and $T$ is the gas temperature. $a$ is defined such that the outflow is only defined when the drift velocity is direct towards the wall and zero otherwise. $r$ is defined as the fraction of particles reflected by the surface. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SecondaryElectronEnergyBC` is defined as

Reviewer: ```suggestion
Where $\Gamma$ is the flux of the electron or ions, $\textbf{n}$ is the normal vector of the boundary,
$\mu$ is the mobility coefficient, $n$ is the species density, $v_\text{th}}$ is the thermal velocity of the species, $k_{B}$ is the Boltzmann constant, and $T$ is the gas temperature. $a$ is defined such that the outflow is non-zero when the drift velocity is directed towards the wall and zero otherwise. $r$ is defined as the fraction of particles reflected by the surface. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SecondaryElectronEnergyBC` is defined as
```

> \begin{equation}
> \Gamma_{\epsilon} \cdot \textbf{n} = \frac{5}{3} \frac{-2}{1+r} (1 - a_{e}) \gamma \Gamma_{j} \cdot \textbf{n} \\[10pt]
> \Gamma_{j} \cdot \textbf{n} = \frac{1-r_{j}}{1+r_{j}} \left[ (2 a_{j}-1) \ \mu_{j} \text{-} \nabla (V / l_{c}) \cdot \textbf{n} \ \exp(N_{j}) + \frac{1}{2}v_{th,j}\exp(N_{j}) \right]
> \end{equation}

Reviewer: ```suggestion
\begin{equation}
\Gamma_{\epsilon} \cdot \textbf{n} = \frac{5}{3} \frac{-2}{1+r} (1 - a_{e}) \gamma \Gamma_{j} \cdot \textbf{n} \\[10pt]
\Gamma_{j} \cdot \textbf{n} = \frac{1-r_{j}}{1+r_{j}} \left[ (2 a_{j}-1) \ \mu_{j} \left( \text{-} \nabla (V / l_{c})  \right) \cdot \textbf{n} \ \exp(N_{j}) + \frac{1}{2}v_{\text{th},j}\exp(N_{j}) \right]
\end{equation}
```

> Where $N_{j}$ is the molar density of the specie in log form and $l_{c}$ is the scaling factor of the mesh.

Reviewer: ```suggestion
Where $N_{j}$ is the molar density of the species in log form and $l_{c}$ is the scaling factor of the mesh.
```

## Example Input File Syntax

> An example of how to use `SecondaryElectronEnergyBC` can be found in the
> test file `mean_en.i`.

Reviewer: ```suggestion
```
